### PR TITLE
VIH-8008 Adding translations

### PR DESCRIPTION
**Pull request by FarronF**
### JIRA link (if applicable) ###
VIH-8008

### Change description ###
Adding translations for solicitor and barrister

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by PrasannaDommalapati — APPROVED** [review]
LGTM

**Review by simonwkelso — APPROVED** [review]
LGTM Farron

File: VideoWeb/VideoWeb/ClientApp/src/assets/i18n/cy.json [hunk 1]
```diff
@@ -941,7 +941,9 @@
     "appellant-lip": "Ymgyfreithiwr drosto'i hun yr Apelydd",
     "app-advocate": "Eiriolwr yr Apelydd",
     "app": "Apelydd",
-    "video-access-point": "Pwynt mynediad fideo"
+    "video-access-point": "Pwynt mynediad fideo",
+    "solicitor": "Cyfreithiwr",
+    "barrister": "Bargyfreithiwr"
   },
   "case-type-group": {
```
Comment (khizarkhanu-hmcts) [inline VideoWeb/VideoWeb/ClientApp/src/assets/i18n/cy.json:948]: LGTM

File: VideoWeb/VideoWeb/ClientApp/src/assets/i18n/en.json [hunk 2]
```diff
@@ -652,6 +653,7 @@
     "respondent-Advocate": "Respondent Advocate",
     "respondent-lip": "Respondent LIP",
     "special-counsel": "Special Counsel",
+    "solicitor": "Solicitor",
     "state-lip": "State LIP",
```
Comment (khizarkhanu-hmcts) [inline VideoWeb/VideoWeb/ClientApp/src/assets/i18n/en.json:657]: LGTM

**Review by khizarkhanu-hmcts — APPROVED** [review]

**Review by omio-ma — APPROVED** [review]

**Review by simonwkelso — APPROVED** [review]
LGTM - all checks passed also.

**Pull request merged**